fluent-plugin-google-cloud.gemspec

```diff
@@ -1,35 +1,35 @@
 Gem::Specification.new do |gem|
   gem.name          = 'fluent-plugin-google-cloud'
-  gem.description   = <<-eos
+  gem.description   = <<-DESCRIPTION
    Fluentd plugins for the Stackdriver Logging API, which will make logs
    viewable in the Stackdriver Logs Viewer and can optionally store them
    in Google Cloud Storage and/or BigQuery.
    This is an official Google Ruby gem.
-eos
+  DESCRIPTION
   gem.summary       = 'fluentd plugins for the Stackdriver Logging API'
   gem.homepage      =
     'https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud'
   gem.license       = 'Apache-2.0'
   gem.version       = '0.12.10'
   gem.authors       = ['Stackdriver Agents Team']
   gem.email         = ['stackdriver-agents@google.com']
-  gem.required_ruby_version = Gem::Requirement.new('>= 2.2')
+  gem.required_ruby_version = Gem::Requirement.new('>= 2.3')
 
   gem.files         = Dir['**/*'].keep_if { |file| File.file?(file) }
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  # Note: In order to update the Fluentd version, please update both here and
+  # NOTE: In order to update the Fluentd version, please update both here and
   # also the fluentd version in
   # https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/config/software/fluentd.rb.
   gem.add_runtime_dependency 'fluentd', '1.13.3'
+  gem.add_runtime_dependency 'google-api-client', '0.30.8'
   gem.add_runtime_dependency 'googleapis-common-protos', '1.3.10'
   gem.add_runtime_dependency 'googleauth', '0.9.0'
-  gem.add_runtime_dependency 'google-api-client', '0.30.8'
   gem.add_runtime_dependency 'google-cloud-logging', '1.6.6'
   gem.add_runtime_dependency 'google-protobuf', '3.20.0'
   gem.add_runtime_dependency 'grpc', '1.45.0'
-  gem.add_runtime_dependency 'json', '2.2.0'
```

Comment:
Just want to check: is this intended upgrade?

Reply:
Yes, I believe this was required by the newer version of Rubocop…